src/components/ground-nav/ground-nav.twig

```diff
@@ -18,23 +18,11 @@
         {{ lead_in }}
       </h2>
       <p class="c-ground-nav__action-title">
-        {% embed '@cloudfour/components/button/button.twig' with {
+        {% include '@cloudfour/components/button/button.twig' with {
           href: link,
           label: title,
-          _ground_nav_icon: icon
+          content_start_icon: icon ?? 'link'
```

Comment:
I recently learned about Twig's "null-coalescing" operator, which seems preferable to `default` in many cases. If this were written as `icon|default('link')`, you would not be able to override it later with `content_start_icon: false`, because `default` will trigger. But using `icon ?? 'link'` says to use `icon` unless it is undefined or null.